Luis F. Salazar

lfsalazarcruz opened the pull request:
This is my PR.

Commit:
This is my PR.

Commit:
Started Bootstrap Project I.

Added two images, a nav bar and some text to the site.'

groov1234 commented:
My man Luis! I always get a laugh at your PR message. "This is my PR." Those are some giant images! I like the overall layout. Some bugs with the image of yourself I see. Let us know if you need any help! Help channel is waiting!!! Carry on!!

Commit:
Major changes

Adjusted my image. Now my image has a fixed size.

Finished adding major elements. Still need some adjustments.

Commit:
Major Changes

Added custom buttons to the nav bar. Also, buttons have a darken effect.

My image is now circle.

I imported the mixin files from the last project to be able to use those mixins.

In general, my web page looks way better.

Commit:
Major adjustments-Complete Redesign

Gave my resume a kind of blog-style. Added different components, such as a Jumbotron, a Card component, a table, etc.

Organized cards into masonry-like columns and added different images to give more information about my interests.

Imported my mixin.less file from yesterday to give my resume a more stylish look (round corners, button-maker mixin, round images, hover effect, etc.)
